docs/source/admin/installguide.rst

```diff
@@ -300,6 +302,26 @@ Once both processes have started up, you can access the Koverse user interface f
 The default username and password are 'admin' and 'admin'. The password can be changed immediately after logging in.
 
 
+High-Availability (HA) and Failover with Apache Ambari
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Ambari offers HA for the Hadoop Infrastructure via the NameNode, after following this guide you will have a Standby NameNode in addition to your existing NameNode along with Journal Nodes.
+
+Prerequistes
+-------------
+- Verify HDFS and Zookeepers are NOT on maintenance mode.
+- Confirm you have three servers in your cluster running at least three Zookeeper Servers
+
+Installation
+-------------
+
+It is recommended to use this `guide <https://docs.cloudera.com/HDPDocuments/Ambari-2.7.5.0/managing-high-availability/content/amb_enable_namenode_high_availability.html>`_ for enabling HA of the NameNode with automatic failover via Ambari
+
+After the install using the wizard you should see a Standby NameNode like so
+
+.. image:: /_static/InstallGuide/image1.png
+
+
```

Comment:
I think this is good, but we should probably add two more HA items for the underlying stack to protect against single node failure:

* YARN Resource Manager:  https://docs.cloudera.com/HDPDocuments/Ambari-2.7.5.0/managing-high-availability/content/amb_enable_resourcemanager_high_availability.html

* Accumulo Master HA:  Not extensively documented.  But if you run two masters, it will pick one as the primary and the other will be a backup. If the primary fails, the backup will take over.  So you just need to have two hosts with Accumulo Master components installed.

Reply:
Oops, left this in the wrong spot, it is referring to the entire "High-Availability (HA) and Failover with Apache Ambari" section